packages/jellyfish-api-core/__tests__/category/loan/getInterest.test.ts

```diff
@@ -0,0 +1,197 @@
+import { LoanMasterNodeRegTestContainer } from './loan_container'
+import BigNumber from 'bignumber.js'
+import { Testing } from '@defichain/jellyfish-testing'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+
+describe('Loan', () => {
+  const container = new LoanMasterNodeRegTestContainer()
+  const testing = Testing.create(container)
+  let interestTSLABlockHeight: number
+  let txUber: string
+
+  async function setup (): Promise<void> {
+    // token setup
+    const collateralAddress = await testing.container.getNewAddress()
```

Comment:
```suggestion
    const collateralAddress = await testing. generateAddress()
```
Like what you wrote in line 19